docs/testing.md

```diff
@@ -24,3 +24,28 @@ files in the `go/` directory tree.
 
 At a minimum, we suggest testing JSON marshalling and unmarshalling
 of the Go language bindings.
+
+## Testing the Python bindings
+
+The `tests/python` directory contains a number of tests for the statement
+and predicate layers.
+
+### Running the Python tests
+
+To run all tests:
+
+```shell
+make py_test
+```
+
+### Writing new Python tests
+
+Please use the standard [Python unittest package] to write tests
+for new predicates. For example tests, take a look at the `test_*.py`
+modules in the `tests/python/` directory tree.
+
+At a minimum, we suggest testing JSON marshalling and unmarshalling
+of the Python language bindings.
+
+[Golang testing package]: https://pkg.go.dev/testing
```

Comment:
Nit: I know you are fixing an existing use of the term, but it's Go _not_ Golang :-)